src/multibody/model.hpp

```diff
@@ -220,7 +224,11 @@ namespace pinocchio
         res.frames[k] = frames[k].template cast<NewScalar>();
       }
       
+      /// TODO: remove this pragma when neutralConfiguration will be removed
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
       return res;
+#pragma GCC diagnostic pop
```

Comment:
Why here? Is `return res;` causing warnings?

Reply:
Maybe this one is not useful. You're certainly right.